src/main/resources/pdl/schema.graphql

```diff
@@ -697,17 +697,22 @@ enum Endringstype {
 }
 
 input Criterion {
-    # Feltnavn ikludert sti til ønsket felt (Eksempel: person.navn.fornavn)
-    fieldName:String!
+    # Feltnavn inkludert sti til ønsket felt (Eksempel: person.navn.fornavn)
+    fieldName:String
+
```

Comment:
Uten «!» - fungerer dette på samme måte?

Reply:
Feltet har blitt nullable, så det skal gå fint. Vi bruker nok ikke denne delen av skjemaet heller, da dette er brukt i søk. Har også deployet til dev for å teste og det ser fint ut.